src/app/header/header.component.html

```diff
@@ -1,74 +1,157 @@
-<nav class="navbar navbar-default navbar-pf" role="navigation" *ngIf="loggedIn">
-    <div class="navbar-header"></div>
+<nav class="navbar navbar-inverse navbar-pf" role="navigation" *ngIf="loggedIn">
```

Comment:
The responsive behavior of the overall menu makes the one hidden when mobile experience is  active and the other when desktop experience is active.  When you run the app .... collapse the width of the browser to mobile size and this menu kicks in ...